### ISSUE TITLE
Arrows are not visible

### ISSUE THREAD

**Issue by leif81** [asciidoctor/asciidoctor-pdf#62]
In my document I have `foo -> bar` and so the arrow gets procesed into `foo → bar`, but the colour of the arrow is white so it is not visible in the pdf.

**Comment by mojavelinux** [comment]
Note that the arrow now appears thanks to the fallback font fix by @chloerei. See https://github.com/asciidoctor/asciidoctor-pdf/commit/11bcf455f39429706ee5890eb54886ee6a01bced

However, I'd still like to update the NotoSerif font file so that it doesn't have to use the fallback for this character.

**Comment by mojavelinux** [comment]
I looked into it and, as it turns out, the single arrows in NotoSerif are really ugly and don't line up evenly in the line. The double arrows are just missing. For those reasons, I actually prefer that the arrows fallback to M+ 1p. Now that we have fallback font support, we can rely on this. I added a note in the default theme that the fallback font is providing these characters.

I wish Prawn would warn when a glyph isn't resolved, but it's silent about it. I may pursue this upstream to see if we can get a warning. See https://github.com/prawnpdf/prawn/blob/master/lib/prawn/text/formatted/box.rb#L427-L434
